src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -467,6 +467,21 @@ public Vector4 CornerRadius
         // Accessibility
         //
 
+        /// <summary>
+        /// Gets or sets a value that allows the automation framework to find and interact with this element.
+        /// </summary>
+        /// This will be public opened in tizen_5.0 after ACR done. Before ACR, need to be hidden as inhouse API.
```

Comment:
please remove this comment~ this comment is not appropriate for this API.
_**/// This will be public opened in tizen_5.0 after ACR done. Before ACR, need to be hidden as inhouse API.**_